From 701f154310a56d052ac99bdc2bdf801dab7810cc Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@notvalid.com>
Date: Mon, 19 Oct 2026 14:47:06 -0700
Subject: [PATCH] test: expect V2's 25% staking reward in the mining RPC tests

v2/042 sets STAKING_REWARD_RATIO to 25. abc_p2p_avalanche_policy_stakingrewards
already uses 25, but abc_mining_basic and abc_mining_stakingrewards still
expected eCash's 10% (gbt stakingrewards amount 625000000 vs 250000000).
Both pass with 25.

Tier: V2 (fixup to v2/042)
---
 test/functional/abc_mining_basic.py          | 3 ++-
 test/functional/abc_mining_stakingrewards.py | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/test/functional/abc_mining_basic.py b/test/functional/abc_mining_basic.py
index aad3b60122..a05330f1b1 100644
--- a/test/functional/abc_mining_basic.py
+++ b/test/functional/abc_mining_basic.py
@@ -19,7 +19,8 @@ MINER_FUND_ADDR = "ecregtest:prfhcnyqnl5cgrnmlfmms675w93ld7mvvq9jcw0zsn"
 MINER_FUND_LEGACY_ADDR = "2NCXTUCFd1Q3EteVpVVDTrBBoKqvMPAoeEn"
 
 MINER_FUND_RATIO = 32
-STAKING_REWARD_RATIO = 10
+# V2 pays 25% of the coinbase to staking rewards (v2/042).
+STAKING_REWARD_RATIO = 25
 
 
 class AbcMiningRPCTest(BitcoinTestFramework):
diff --git a/test/functional/abc_mining_stakingrewards.py b/test/functional/abc_mining_stakingrewards.py
index 37e161cd62..4bc401e757 100644
--- a/test/functional/abc_mining_stakingrewards.py
+++ b/test/functional/abc_mining_stakingrewards.py
@@ -23,7 +23,8 @@ from test_framework.util import (
 )
 
 QUORUM_NODE_COUNT = 16
-STAKING_REWARDS_COINBASE_RATIO_PERCENT = 10
+# V2 pays 25% of the coinbase to staking rewards (v2/042).
+STAKING_REWARDS_COINBASE_RATIO_PERCENT = 25
 
 
 class LongpollThread(threading.Thread):
-- 
2.50.1 (Apple Git-155)

